back-end/e-learning/app/Http/Controllers/CategoryController.php

```diff
@@ -56,4 +56,17 @@ public function update(Request $request, $id){
 			'message' => 'ID not found!'
 		], 404); 
 	}
+	
+	public function remove($id){
+		$category = Category::find($id);
+		if($category){
```

Comment:
Put space before `{` when writing block conditional statements

```suggestion
		if($category) {
```